vars/updatecli.groovy

```diff
@@ -4,7 +4,8 @@ def call(userConfig = [:]) {
     action: 'diff',
     config: './updatecli/updatecli.d',
     values: './updatecli/values.yaml',
-    updatecliDockerImage: 'ghcr.io/updatecli/updatecli:v0.16.0',
+    updatecliDockerImage: 'ghcr.io/updatecli/updatecli:latest',
```

Comment:
@lemeurherve can you make sure that your branch is rebased against the master branch?